docs/concepts/cluster-administration/device-plugins.md

```diff
@@ -48,9 +48,12 @@ to advertise 2 `vendor-domain/foo`.
 Then, developers can request devices in a
 [Container](/docs/api-reference/{{page.version}}/#container-v1-core)
 specification by using the same process that is used for
-[opaque integer resources](/docs/tasks/configure-pod-container/opaque-integer-resource/).
-In version 1.8, extended resources are supported only as integer resources and must have
-`limit` equal to `request` in the Container specification.
+[opaque integer resources](/docs/tasks/configure-pod-container/opaque-integer-resource/),
+with the following limitations:
+  * As of now, extended resources are supported only as integer resources and must have
+    `limit` equal to `request` in the Container specification.
+  * Devices can NOT be shared among Containers.
+  * We currently do NOT support requesting device plugin resources in [Init Containers](/docs/concepts/workloads/pods/init-containers/).
```

Comment:
Or `We currently do NOT support sharing device plugin resources between [Init Containers] and regular containers` ?

Reply:
There is also a possibility of reuse between init-containers, you could take a look at the [comment](https://github.com/kubernetes/kubernetes/issues/56022#issuecomment-348103280) :)